src/main/java/fruit/shop/service/strategy/BalanceHandler.java

```diff
@@ -0,0 +1,8 @@
+package fruit.shop.service.strategy;
+
+public class BalanceHandler implements ActivityHandler {
+    @Override
+    public int getOptionResult(int currentValue, int addValue) {
+        return addValue;
+    }
+}

```

Comment:
BalanceOperationHandler should look like this:
```
public class BalanceOperationHandlerImpl implements OperationHandler {
    @Override
    public void handle(FruitTransaction fruitTransaction) {
        Storage.FRUITS.put(fruitTransaction.getFruit(), fruitTransaction.getQuantity());
    }
}
```
The same should be for other handlers